sdk/storage/Azure.Storage.Blobs/src/BlobContainerClient.cs

```diff
@@ -81,6 +82,16 @@ public class BlobContainerClient
         /// </summary>
         internal virtual CustomerProvidedKey? CustomerProvidedKey => _customerProvidedKey;
 
+        /// <summary>
+        /// The <see cref="EncryptionScope"/> to be used when sending requests.
+        /// </summary>
+        internal readonly EncryptionScope _encryptionScope;
```

Comment:
I'm leaning toward removing this property from BlobContainer, it isn't actually used on any operations, and I think users may find the ambiguity between the BlobContainerClient.Create() ContainerEncryptionScopeOptions parameter and this property confusing.

Reply:
> I'm leaning toward removing this property from BlobContainer, it isn't actually used on any operations, and I think users may find the ambiguity between the BlobContainerClient.Create() ContainerEncryptionScopeOptions parameter and this property confusing.

The issue I see with this is that the BlobClientOptions property would have no effect at all for Containers.